services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/volumes_removal.py

```diff
@@ -0,0 +1,22 @@
+from models_library.projects_nodes_io import NodeID
+from servicelib.rabbitmq._client_rpc import RabbitMQRPCClient
+from servicelib.rabbitmq.rpc_interfaces.agent.volumes import (
+    remove_volumes_without_backup_for_service,
+)
+
+
+async def remove_volumes_from_node(
+    rabbit_rpc_client: RabbitMQRPCClient,
+    docker_node_id: str,
+    swarm_stack_name: str,
+    *,
+    node_id: NodeID,
+) -> None:
+    """removes all service volumes form the node where it was running"""
+
+    await remove_volumes_without_backup_for_service(
```

Comment:
Q: what is the purpose of creating a full new module for a one-to-one function? 
NOte that here you can literaly make
```python

remove_volumes_from_node = remove_volumes_without_backup_for_service
```